pkgs/development/python-modules/openstep-parser/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "openstep-parser";
+  version = "2.0.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "kronenthaler";
+    repo = "openstep-parser";
+    rev = "refs/tags/2.0.0";
+    hash = "sha256-xBxbSKiF6zVTwZGGjPADx1LkhQjX2VHZsct/xf3rkTs=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "openstep_parser" ];
+
+  meta = with lib; {
+    description = "OpenStep plist reader/writer for Python";
+    homepage = "https://github.com/kronenthaler/openstep-parser";
+    changelog = "https://github.com/kronenthaler/openstep-parser/blob/master/CHANGELOG.md";
+    license = licenses.bsd3;
```

```suggestion
    license = licenses.bsd2;
```
https://github.com/kronenthaler/openstep-parser/blob/2.0.0/LICENSE